### ISSUE TITLE
http-terminator: Please verify that the package.json has a valid "main" entry.

### ISSUE THREAD

**Issue by fmoessle** [gajus/lightship#39]
Node version: v14.17.0
Lightship version: 6.7.2
OS: macOS 11.5.2

```typescript
import { createLightship } from 'lightship'

const lightship = createLightship()
```

Running my application throws this error:
```console
[ERROR] 19:03:22 Error: Cannot find module '/Users/project/node_modules/http-terminator/dist/src/index.js'. 
Please verify that the package.json has a valid "main" entry
```

Maybe the error comes from the last update in http-terminator 3.0.1: https://github.com/gajus/http-terminator/releases/tag/v3.0.1

**Comment by fmoessle** [comment]
Fixing the sub-dependency in package.json temporarily solves the issue:

```JSON
{
  "dependencies": {
    "lightship": "6.7.2"
  },
  "resolutions": {
    "http-terminator": "3.0.0"
  }
}
```

**Comment by fmoessle** [comment]
Solved with http-terminator 3.0.3.
